--- method_index_2015.rb
+++ method_index_2014.rb
@@ -32,7 +32,6 @@
 add_action_callback
 add_arc
 add_circle
-add_classification
 add_cline
 add_context_menu_handler
 add_cpoint
@@ -101,9 +100,7 @@
 casts_shadows=
 casts_shadows?
 center
-center_2d
 check
-classifications
 classify_point
 clear
 clear!
@@ -114,9 +111,6 @@
 cm
 color
 color=
-colorize_deltas
-colorize_type
-colorize_type=
 commit_operation
 common_edge
 common_face
@@ -143,7 +137,6 @@
 curve_interior?
 cuts_opening=
 cuts_opening?
-days_remaining
 deactivate
 definition
 definition=
@@ -205,7 +198,6 @@
 erase
 erase!
 erase_entities
-error_description
 execute_script
 expectsStartupModelNotifications
 explode
@@ -223,7 +215,6 @@
 file_new
 filename
 fill_from_mesh
-find_entity_by_id
 find_faces
 find_support_file
 find_support_files
@@ -241,7 +232,6 @@
 format_length
 fov
 fov=
-fov_is_height?
 georeferenced?
 getExtents
 getInstructorContentDirectory
@@ -249,12 +239,10 @@
 get_UVHelper
 get_attribute
 get_back_UVQ
-get_classification_value
 get_datfile_info
 get_datum
 get_default_dialog_color
 get_element_value
-get_extension_license
 get_front_UVQ
 get_glued_instances
 get_i18ndatfile_info
@@ -316,8 +304,6 @@
 invert!
 is2d=
 is2d?
-is_2d?
-is_64bit?
 is_curve?
 is_online
 is_polygon?
@@ -345,7 +331,6 @@
 leaf_at
 length
 length=
-licensed?
 line
 line_stipple=
 line_weight
@@ -357,7 +342,6 @@
 load_file
 load_from_url
 load_on_start?
-load_schema
 loaded?
 local_bounds
 lock_inference
@@ -399,8 +383,6 @@
 move_vertices
 name
 name=
-names
-namespace
 navigation_buttons_enabled=
 navigation_buttons_enabled?
 new
@@ -418,7 +400,6 @@
 offset!
 offset_vector
 offset_vector=
-onActivateModel
 onActivePathChanged
 onActiveSectionPlaneChanged
 onActiveToolChanged
@@ -533,7 +514,6 @@
 pixelwidth
 place_component
 plane
-planeplatform
 platform
 play_sound
 plugins_disabled=
@@ -577,7 +557,6 @@
 register_importer
 registered?
 remove
-remove_classification
 remove_frame_change_observer
 remove_observer
 rendering_options
@@ -596,11 +575,9 @@
 save_copy
 save_thumbnail
 savepanel
-scale_2d
 scaling
 screen_coords
 screen_scale_factor
-select_directory
 select_tool
 selected_page
 selected_page=
@@ -612,7 +589,6 @@
 set!
 set_attribute
 set_background_color
-set_classification_value
 set_color_from_line
 set_cursor
 set_datum
@@ -663,7 +639,6 @@
 start_operation
 start_timer
 start_vertex_normal
-state
 status_bar_text
 status_bar_text=
 status_text=
@@ -732,7 +707,6 @@
 union
 unique_name
 unitvector?
-unload_schema
 up
 update
 update_selected_style
@@ -756,7 +730,6 @@
 used_by?
 utm_to_point
 uv_at
-uvs
 valid?
 values
 vcb_label=
